### ISSUE TITLE
Run frontend and backend servers in parallel

### ISSUE THREAD

**Issue by rmallols** [rmallols/kodflix#21]
We still need two commands to startup frontend and backend servers:
- frontend: `npm start`.
- backend: `npm start-backend`.

We want to start both servers using a single command using `npm`.

**Comment by rmallols** [comment]
**Technical guidance:**

1. Install [npm-run-all](https://www.npmjs.com/package/npm-run-all)

2. Create a new task `start-all` to run both npm tasks in parallel: `npm-run-all -p <task1> <task2>`
